### PR TITLE
Implement Stats API fn

### PR DESCRIPTION
**Pull request by Clebal**
I implemented a fn to call the Stats API. In order to make the Stats API available also for static tilesets I implemented getTileJson which makes a Maps API request.

### CODE REVIEW

File: packages/react-api/src/api/stats.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * @param { object } props
+ * @param { string } props.column -
+ * @param { object } props.source -
+ * @param { object= } props.opts - Additional options for the HTTP request
+ */
+export async function callStats(props) {
+  mandatoryProps.forEach((prop) => {
+    if (!props[prop]) throw new Error('callStats: Missing source');
+  });
```
Comment (zbigg) [inline packages/react-api/src/api/stats.js:17]: for missing `column` it complains 'missing source'

Anyway, i would propose that "explicit" is more readable, more performant and ... even shorter:

```
assert(props.source, 'callStats: missing source')
assert(props. column, 'callStats: missing column')
```

**Review by zbigg — APPROVED** [review]
One minor nitpick, otherwise LGTM.

File: packages/react-api/src/api/lds.js [hunk 2]
```diff
@@ -29,30 +28,11 @@ export async function ldsGeocode({ credentials, address, country, limit, opts })
...
-      headers: {
-        Authorization: `Bearer ${credentials.accessToken}`
-      },
-      signal: abortController?.signal,
-      ...otherOptions
-    });
-    data = await response.json();
-  } catch (error) {
-    if (error.name === 'AbortError') throw error;
+  let data = await makeCall({ url, credentials, opts });
```
Comment (VictorVelarde) [inline packages/react-api/src/api/lds.js:31]: Nice refactor

File: packages/react-api/__tests__/api/stats.test.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+const TABLE_AND_QUERY_STATS = {
+  attribute: 'injuries',
+  type: 'Number',
+  avg: 0.44776268477826375,
+  sum: 4483,
+  min: 0,
+  max: 7,
+  quantiles: {
+    3: [0, 7],
```
Comment (VictorVelarde) [inline packages/react-api/__tests__/api/stats.test.js:11]: Nice piramid 🔺 !

Comment (Clebal) [reply packages/react-api/__tests__/api/stats.test.js:11]: It's very beautiful hahaha.

**Review by zbigg — APPROVED** [review]

**Review by VictorVelarde — APPROVED** [review]
🚀 . You're free to go @Clebal

**Pull request merged**